### PR TITLE
OCI source: return error if open blob failed

### PR DESCRIPTION
**Pull request by hustcat**
OCI source: return error if open blob failed. This is needed by [here](https://github.com/containers/image/blob/master/image/docker_schema2.go#L85)

Signed-off-by: Ye Yin <eyniy@qq.com>

### CODE REVIEW

File: types/types.go [hunk 1]
```diff
@@ -186,7 +186,7 @@ type Image interface {
 	// ConfigInfo returns a complete BlobInfo for the separate config object, or a BlobInfo{Digest:""} if there isn't a separate object.
 	// Note that the config object may not exist in the underlying storage in the return value of UpdatedImage! Use ConfigBlob() below.
 	ConfigInfo() BlobInfo
-	// ConfigBlob returns the blob described by ConfigInfo, iff ConfigInfo().Digest != ""; nil otherwise.
+	// ConfigBlob returns the blob described by ConfigInfo, if ConfigInfo().Digest != ""; nil otherwise.
```
Comment (mtrmac) [inline types/types.go]: This is an unrelated change, and `iff` is used intentionally ( https://en.wikipedia.org/wiki/If_and_only_if ). Please drop.

**Comment by mtrmac** [comment]
:+1:

**Comment by runcom** [comment]
lgtm
